azure-pipelines.yml

```diff
@@ -4,11 +4,20 @@
 # https://aka.ms/yaml
 
 trigger:
-- 64097-GCP-hosting
+- 6666-automatic-gcp-deployment
 
 pool:
   vmImage: ubuntu-latest
 
+variables:
+  icone_project_name: 'cdot-oim-wzdx-dev'
+  cotrip_project_name: 'cdot-oim-wzdx-prod'
+  navjoy_project_name: 'cdot-oim-wzdx-dev'
+
+  icone_function_name: 'auto_icone_translator_ftp'
+  cotrip_function_name: 'salesforce-data'
+  navjoy_function_name: 'navjoy-568-translator'
+
 steps:
 
 - task: DownloadSecureFile@1
```

Comment:
Can we add some description for this file either to the existing README or another documentation page? It would be nice to have some text describing what we're doing in this file, specifically how this GoogleServiceAccountKey.json lives in devops and is injected into the pipeline (and how we could update that if needed).